patches/minecraft/net/minecraft/block/BlockDynamicLiquid.java.patch

```diff
@@ -17,3 +17,17 @@
                      p_176375_3_.func_177230_c().func_176226_b(p_176375_1_, p_176375_2_, p_176375_3_, 0);
                  }
              }
+@@ -254,11 +255,12 @@
+ 
+     private boolean func_176372_g(World p_176372_1_, BlockPos p_176372_2_, IBlockState p_176372_3_)
+     {
++        Material material = p_176372_1_.func_180495_p(p_176372_2_).func_185904_a();
```

Comment:
Please move this line down so the two patched lines are right next to each other